R/stations.R

```diff
@@ -278,7 +278,7 @@ stations_dl_internal <- function(skip = NULL, verbose = FALSE, quiet = FALSE,
 #' @param coords Numeric. A vector of length 2 with latitude and longitude of a
 #'   place to match against. Overrides \code{lat} and \code{lon} if also
 #'   provided.
-#' @param dist Numeric. Match all stations within this many kilometres of the
+#' @param dist Numeric. Match all stations within this many kilometers of the
```

Comment:
I'd prefer to keep with Canadian spelling :wink:
```suggestion
#' @param dist Numeric. Match all stations within this many kilometres of the
```

Reply:
Ok! :smile: